mitodl/mit-open
# rename MIT Open to MIT Learn

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/5161
Addresses "Site Content" items in https://github.com/mitodl/hq/issues/5154

### Description (What does it do?)
This PR replaces instances of "MIT Open" and `mit-open` and other variations on the terminology with "MIT Learn" and `mit-learn` respectively.

### Screenshots (if appropriate):
![image](https://github.com/user-attachments/assets/bc126970-b626-4886-9b0d-fc849d59db6b)
![image](https://github.com/user-attachments/assets/c0028a14-3b1c-4d1a-83a5-46a31f7942ef)

### How can this be tested?
 - Switch to this branch and "clean slate" your setup to ensure you don't have any stale artifacts:
   - Delete all `node_modules` folders
   - Delete `frontends/mit-open` (it's been renamed to `frontends/mit-learn`, so after switching to this branch it may leave behind untracked folders such as `bulid`)
   - Rebuild Docker containers
 - Spin up `mit-learn`
 - Smoke test the site and make sure there are no references to "MIT Open" or Lorem Ipsum text where there should be the new name / logo
